management-service/services/experiment_service_test.go

```diff
@@ -163,28 +163,30 @@ func testListExperiments(s *ExperimentServiceTestSuite) {
 	stringSegmenter := []string{"seg-1"}
 	boolSegmenter := []string{"true"}
 
+	var nilPagingResponse *pagination.Paging
+
 	// All experiments under a settings
-	expResponsesList, pagingResponse, err := svc.ListExperiments(1, services.ListExperimentsParams{})
+	actualResponsesList, pagingResponse, err := svc.ListExperiments(1, services.ListExperimentsParams{})
```

Comment:
Renamed this as it's clearly the actual response, and not the expected resopnse.